src/responses.rs

```diff
@@ -13,3 +13,17 @@ pub fn merge_responses(responses: Vec<Response>) -> Response {
     }
     merged
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_merge_empty_responses() {
```

Comment:
add test_case for non empty case

Reply:
good catch. don't know what is contained in the binary tbh. lets bring it up during standup to